src/app/_components/client/Button/Button.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import React, { MouseEvent } from "react";
+
+import { ButtonProps } from "./Button.type";
+import { useConvertButtonStyle } from "./hooks";
+
+import { motion } from "framer-motion";
+import { twJoin, twMerge } from "tailwind-merge";
+
+const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+  (
+    {
+      children,
+      radius = "0.94rem",
+      fontSize = "0.94rem",
+      fontWeight = "bold",
+      fontColor = "black",
+      backgroundColor = "yellow",
+      border = "none",
+      shadow = false,
+      className,
+      onClick,
+      disabled = false,
+      ...rest
+    },
+    ref,
+  ) => {
+    const convertedStyle = useConvertButtonStyle({
+      radius,
+      fontSize,
+      fontWeight,
+      fontColor,
+      backgroundColor,
+      border,
+      shadow,
+    });
+
+    const handleClick = (e: MouseEvent<HTMLButtonElement>) => {
+      e.preventDefault();
+      if (!onClick || disabled) return;
```

Comment:
[ P5 ] 

Convention에 아래와 같은 규칙이었던것 같습니다!

```suggestion
      if (!onClick || disabled) {
        return 
      };
```

Reply:
> 헉 습관적으로 저렇게 썼네요 !! 수정해두겠습니다 ㅎㅎ